Spatial_searching/doc/Spatial_searching/CGAL/Fuzzy_sphere.h

```diff
@@ -3,25 +3,33 @@ namespace CGAL {
 /*!
 \ingroup RangeQueryItemClasses
 
-The class `Fuzzy_sphere` implements fuzzy `d`-dimensional spheres. 
-A fuzzy sphere with radius \f$ r\f$ and fuzziness value \f$ \epsilon\f$ has 
-as outer approximation a sphere with radius \f$ r+\epsilon\f$ and 
-as inner approximation a sphere with radius \f$ r-\epsilon\f$. 
-
-
-\tparam Traits must be a model of the concept 
-`SearchTraits`, for example `CGAL::Cartesian_d<double>`. 
+The class `Fuzzy_sphere` implements fuzzy `d`-dimensional spheres.
+A fuzzy sphere with radius \f$ r\f$ and fuzziness value \f$ \epsilon\f$ has
+as inner approximation a sphere with radius \f$ r-\epsilon\f$ and
+as outer approximation a sphere with radius \f$ r+\epsilon\f$.
+
+\attention The fuzziness of a `Fuzzy_sphere` is specified by a parameter \f$ \epsilon\f$
+denoting a maximal allowed distance to the boundary of a sphere.
+If the distance to the boundary is at least \f$ \epsilon\f$, points inside the
+object are always reported and points outside the sphere are never reported.
+Points within distance \f$ \epsilon\f$ to the boundary may be or may be not reported.
+Subsequently, points on the inner and outer spheres may or may not be reported
+by a search query. Specifically when \f$ \epsilon = 0\f$, points on the sphere
+of radius \f$ r\f$ may or may not be reported.
```

Comment:
The previous paragraph is some remnant that I had rephrased to use less/greater... but is somehow there without these changes. There were many "distance of at least.... distance of at most... distance within" which were quite confusing; should be good now.